scripts/generate_specs.sh

```diff
@@ -4,13 +4,13 @@ set -e
 
 parachain_id=$1
 build_step=$2
-
+profile=$3
 if [[ $parachain_id == "" ]]; then
   echo "Chain Name or Parachain ID argument not provided"
```

Comment:
@wilwade  here the profile is passed as parameters and makefile/spec job has been updated